shared/fs/files-loading-hoc.js

```diff
@@ -41,11 +41,7 @@ const FilesLoadingHoc = (ComposedComponent: React.ComponentType<any>) =>
   class extends React.PureComponent<FilesLoadingHocProps> {
     _load = () => {
       const pathLevel = Types.getPathLevel(this.props.path)
-      if (pathLevel < 2) {
-        return
-      }
-      pathLevel === 2 && this.props.loadFavorites()
-      this.props.loadFolderList()
+      pathLevel > 2 ? this.props.loadFolderList() : pathLevel === 2 && this.props.loadFavorites()
```

Comment:
did you intend to no longer `loadFolderList` for `pathLevel === 2`?

Reply:
Intended but by mistake. That seemed unnecessary at this time, but then I realized later that it was actually needed, so added it back here: https://github.com/keybase/client/pull/13428/files#diff-ebef47f5a7fc7af84113ea74db58772bR48